From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 16 Jan 2020 12:59:08 -0800
Subject: [PATCH] Allow unknown options in configure script.

---
 configure | 2 --

--- qemu-7.1.0/configure~	2022-08-30 16:41:57.000000000 +0000
+++ qemu-7.1.0/configure	2022-08-31 16:35:16.320377997 +0000
@@ -703,7 +703,6 @@
   if test $? -eq 1; then
     echo "ERROR: unknown option $1"
     echo "Try '$0 --help' for more information"
-    exit 1
   fi
 }
 
