From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 16 Jan 2020 12:59:08 -0800
Subject: [PATCH] Allow unknown options in configure script.

---
 configure | 2 --
 1 file changed, 2 deletions(-)

diff --git a/configure b/configure
index 48c21775f..32353c4ff 100755
--- a/configure
+++ b/configure
@@ -734,7 +734,6 @@ meson_option_parse() {
   if test $? -eq 1; then
     echo "ERROR: unknown option $1"
     echo "Try '$0 --help' for more information"
-    exit 1
   fi
 }
 
@@ -1200,7 +1199,6 @@ for opt do
   *)
       echo "ERROR: unknown option $opt"
       echo "Try '$0 --help' for more information"
-      exit 1
   ;;
   esac
 done
