From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 16 Jan 2020 12:59:08 -0800
Subject: [PATCH] Allow unknown options in configure script.

---
 configure | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configure b/configure
index 18c26e038..37b8bb021 100755
--- a/configure
+++ b/configure
@@ -1528,7 +1528,6 @@ for opt do
   *)
       echo "ERROR: unknown option $opt"
       echo "Try '$0 --help' for more information"
-      exit 1
   ;;
   esac
 done
