From 73c55d9e0d46625ce0dd3a4590572f66dc2d8301 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Mon, 4 May 2015 17:59:41 -0500
Subject: [PATCH] Use /run/lock

---
 qemu-nbd.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/qemu-nbd.c b/qemu-nbd.c
index 27164b82..cd162281 100644
--- a/qemu-nbd.c
+++ b/qemu-nbd.c
@@ -42,7 +42,7 @@
 #include "trace/control.h"
 #include "qemu-version.h"
 
-#define SOCKET_PATH                "/var/lock/qemu-nbd-%s"
+#define SOCKET_PATH                "/run/lock/qemu-nbd-%s"
 #define QEMU_NBD_OPT_CACHE         256
 #define QEMU_NBD_OPT_AIO           257
 #define QEMU_NBD_OPT_DISCARD       258
-- 
2.14.1

