From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 16 Jan 2020 13:02:51 -0800
Subject: [PATCH] Set default number of sockets to 1.

---
 hw/core/machine.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/core/machine.c b/hw/core/machine.c
index d0408049b..1586774c5 100644
--- a/hw/core/machine.c
+++ b/hw/core/machine.c
@@ -700,7 +700,7 @@ static void smp_parse(MachineState *ms, QemuOpts *opts)
 {
     if (opts) {
         unsigned cpus    = qemu_opt_get_number(opts, "cpus", 0);
-        unsigned sockets = qemu_opt_get_number(opts, "sockets", 0);
+        unsigned sockets = qemu_opt_get_number(opts, "sockets", 1);
         unsigned cores   = qemu_opt_get_number(opts, "cores", 0);
         unsigned threads = qemu_opt_get_number(opts, "threads", 0);
 
