From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Auke Kok <auke-jan.h.kok@intel.com>
Date: Thu, 16 Jan 2020 13:02:51 -0800
Subject: [PATCH] Set default number of sockets to 1.

---
 hw/core/machine.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/core/machine.c b/hw/core/machine.c
index d0408049b..1586774c5 100644
--- a/hw/core/machine.c
+++ b/hw/core/machine.c
@@ -746,7 +746,7 @@ void machine_set_cpu_numa_node(MachineState *machine,
 static void smp_parse(MachineState *ms, SMPConfiguration *config, Error **errp)
 {
     unsigned cpus    = config->has_cpus ? config->cpus : 0;
-    unsigned sockets = config->has_sockets ? config->sockets : 0;
+    unsigned sockets = config->has_sockets ? config->sockets : 1;
     unsigned cores   = config->has_cores ? config->cores : 0;
     unsigned threads = config->has_threads ? config->threads : 0;
 
