--- qemu-2.2.1/vl.c~	2015-03-10 13:38:32.000000000 -0400
+++ qemu-2.2.1/vl.c	2015-04-11 16:37:57.543560860 -0400
@@ -1273,8 +1273,8 @@
 
         unsigned cpus    = qemu_opt_get_number(opts, "cpus", 0);
         unsigned sockets = qemu_opt_get_number(opts, "sockets", 0);
-        unsigned cores   = qemu_opt_get_number(opts, "cores", 0);
-        unsigned threads = qemu_opt_get_number(opts, "threads", 0);
+        unsigned cores   = qemu_opt_get_number(opts, "cores", 2);
+        unsigned threads = qemu_opt_get_number(opts, "threads", 2);
 
         /* compute missing values, prefer sockets over cores over threads */
         if (cpus == 0 || sockets == 0) {
